static/index.html

```diff
@@ -66,15 +65,27 @@ <h1>Conference about Open Source</h1>
             </form>
           </div>
         </div>
-        </div>
       </div>
     </header>
-      <section class ="content">
-       <div class="container">
+    <section class ="content">
+      <div class = "content-wrapper">
+        <div class="container">
           <div class="row">
+            <div class = "content-wrapper2 col-lg-12">
+              <h2>Sponsors</h2>
+                <p>Your contribution will help lower ticket prices for attendees, and as a sponsor, there is something in it for you</p>
+              <div class = "button">  
+                <button type="button" class ="sponsor-button">Sponsor Now</button>
+              </div>
+              <h4>Gold Sponsors</h4>
+              <div class ="sponsors-image">
+                <img src="images/Promet_Logo_Lockup.png">
```

Comment:
This needs a "alt". This will help the site being accessible (users would are blind or with disability ... etc).